pages/index.js

```diff
@@ -1,7 +1,185 @@
-export default function Home() {
+import Triangle from '../components/svg/Triangle';
+import Image from 'next/image';
+import Link from 'next/link';
+import Service from '../components/Service';
+import WhatWeDo from '../components/WhatWeDo';
+import BlogLink from '../components/BlogLink';
+
+export default function home() {
 	return (
 		<main>
-			<h2>Resume</h2>
+			<div className='index--wrapper'>
```

Comment:
The naming thing. This is why I try not to write the page components inside the files. I get you can't name the component index. This is probably the best choice to name the block index, but it still slightly bothers me that the block name doesn't match the function name and that doesn't match the file name. I get it. Not something you need to fix for this PR but you might consider a refactor to pull these page components into a separate file so you can keep with naming conventions.

Reply:
Again _not something you need to fix for this PR_.